examples/03_connectivity/plot_atlas_comparison.py

```diff
@@ -0,0 +1,139 @@
+"""
+Comparing connectomes on different reference atlases
+====================================================
+
+An important part of the example, is to turn a parcellation into
```

Comment:
I would rephrase this as follows: "Shows how to turn a parcellation into a connectome for visualization. ..."